atd-etl/afd_ems_import/requirements.txt

```diff
@@ -0,0 +1,11 @@
+onepasswordconnectsdk==1.*
+psycopg2-binary==2.*
+boto3==1.*
+sshtunnel==0.*
+openpyxl==3.*
+
+# The following two specific versions are needed. 
+# They are borrowed from the dockerized agent vz setup.
+# numpy broke its API in 1.24 wrt to type definitions.
+numpy==1.23.5
+pandas==1.4.4
```

Comment:
eek—can you link to an an issue related to this and/or where this impacts the script? how will we know if/when we can upgrade these packages?

Reply:
@johnclary I'm baffled on this one. I remember working on why these scripts would not run for a non-trivial amount of time, and only after pinning them back to the versions that had been in the docker image they previous ran on fixing it. 

I've unpinned them.

The best I can figure, one of the following is true:

* I'm totally out to lunch and was confused.
* There was something anomalous in the data for EMS or AFD during the time I developed this PR.
* There has been a more recent version of the libraries which addressed the problem that I was encountering.

But it's working now, and I'm utterly unable to recreate this problem. I wish that I had written up an issue at the time so that I knew what had happened.

Thanks for asking for more details!

Reply:
That's great news—thanks for retesting that! ✨ ⬆️ 🍄